javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,43 @@
+const text1=document.getElementById('searchedWord');
+const number1=document.getElementById('noOfImages');
+const ulTag=document.createElement('ul');
+const divTag=document.getElementById('images');
+
+document.getElementById('submit').addEventListener('click',displayGif);
+
+number1.addEventListener('keyup',displayGif);
+
+function displayGif() {
+document.getElementById('message').innerHTML=' ';
```

Comment:
please use a proper code formatter, like `prettier` for VSCode. It makes it easier to grade and also detect some errors in your code.

Reply:
done formatting